hhxx/bit.hpp

```diff
@@ -7,6 +7,7 @@
 
 #include <bitset>
 #include <type_traits>
+#include <climits>
```

Comment:
By the convention used, it should be

~~~C++
#include <climits>

#include <bitset>
#include <type_traits>
~~~